File: src/08-codegolf-medium/network-cabling-codesize/solution.py
n=int(input());X=[];Y=[]
for _ in[0]*n:x,y=map(int,input().split());X+=[x];Y+=[y]
Y.sort();m=Y[n//2];print(max(X)-min(X)+sum(abs(y-m)for y in Y))

File: src/08-codegolf-medium/network-cabling-codesize/solution.cpp
#include<cstdio>
#include<algorithm>
using namespace std;int Y[100001];int main(){int n,x,a=2e9,b=-2e9;scanf("%d",&n);for(int i=0;i<n;i++){scanf("%d%d",&x,&Y[i]);if(x<a)a=x;if(x>b)b=x;}sort(Y,Y+n);long long s=b-a,m=Y[n/2];for(int i=0;i<n;i++)s+=abs(Y[i]-m);printf("%lld",s);}